client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -81,7 +81,12 @@ export default Vue.extend({
       this.frame = frame;
       this.syncedFrame = frame;
       this.emitFrame();
-      this.cacheImages();
+      const imgs = await this.cacheImages();
+      // reset dimensions of map from new image
+      const img = await imgs[this.frame];
+      this.width = img.naturalWidth;
+      this.height = img.naturalHeight;
+      this.resetMapDimensions();
```

Comment:
I had an assumption that resizing the viewing area may be a bit taxing.  Timing this out this function typically takes 1-2ms long to occur.  I think we should filter either inside of `this.resetMapDimensions(width, height)` or elsewhere to make it so it only occurs when the width/height change for the image.  I don't think we want this introducing lag on uniform image-sequences.  I did testing over the EHU dataset running at 60FPS, it will lower the playback speed if trying to `resetMapDimensions` is called on every frame by an average of 1.5 or so ms over the full length of EHU.